transports/webrtc-utils/src/fingerprint.rs

```diff
@@ -34,7 +38,7 @@ pub struct Fingerprint([u8; 32]);
 impl Fingerprint {
     pub(crate) const FF: Fingerprint = Fingerprint([0xFF; 32]);
 
-    #[cfg(test)]
+    // #[cfg(test)]
```

Comment:
Fix me.

Reply:
Good catch, this is a conversion which should be a `From` anyway. Fixed.